### DISCUSSION TITLE
关于 DDLC 中文部及 ICU 论坛的收录问题

### DISCUSSION THREAD

**Discussion by imgradeone** [imgradeone/revolution#9] [DDLC 相关网站投稿 / DDLC Related Website]
由于目前 ddlc.top 域名无法正常访问，我目前已经将其从 [DDLC 相关网站](https://revolution.dokimod.cn/related/) 页面移除。

其次，望 ICU 论坛站方能尽快修复访问主域名就跳转异常页面的劫持问题。虽然 DokiMod 页面已经做了一些 workaround，但此类问题一样会对网站收录造成影响。

**Comment by imgradeone** [comment]
ICU 论坛站方看来已经完成修复了。

**Comment by mtr-static-official** [comment]
这里是ICU 论坛站方。ICU论坛使用的discuz!X性能与安全性过差，论坛方决定逐步的迁移到Monika Forum。你可以在ICU论坛处添加提醒。ICU论坛在本年暂时不会关停。
@imgradeone
